Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -161,8 +161,8 @@
   name: Kammerer
   parent: [BaseWeaponShotgun, BaseGunWieldable, BaseRestrictedContraband]
   id: WeaponShotgunKammerer
-  description: When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds. Uses .50 shotgun shells.
-  components:
+  description: And old design yet a faithful favourite weapon of security forces throughout many worlds. Uses .50 shotgun shells.
```

The original phrasing is a little weird. Plus you misspelled some stuff. 

It kinda feels weird to change the lore from it being the reliable "irregular forces gun" (like the AK-47) to just being another "favorite weapon of security."

```suggestion
  description: An old yet faithful design, and a favorite weapon among irregular forces throughout many worlds. Uses .50 shotgun shells.
```